src/Components/Tabs/ResponsiveTabs.tsx

```diff
@@ -0,0 +1,101 @@
+// Libraries
+import React, {forwardRef, useState} from 'react'
+import classnames from 'classnames'
+
+// Types
+import {
+  ComponentSize,
+  StandardFunctionProps,
+  Alignment,
+  IconFont,
+} from '../../Types'
+
+// Constants
+import {GRID_BREAKPOINT_SM} from '../../Constants'
+
+// Components
+import {Icon} from '../Icon'
+import {ClickOutside} from '../ClickOutside/ClickOutside'
+
+export interface ResponsiveTabsProps extends StandardFunctionProps {
+  /** Size of tabs */
+  size?: ComponentSize
+  /** Alignment of tabs within container (small displays) */
+  mobileAlignment?: Alignment
```

Comment:
`alignmentXS`, `alignmentSM`, etc would be more in line with our existing breakpoint language and doesn't draw an artificial line between devices.